.circleci/golden/expected/src/OpenAPI/Common.hs

```diff
@@ -228,6 +240,11 @@ createBaseRequest config method path queryParams =
         else basePath
     -- filters all maybe
     query = BF.second pure <$> serializeQueryParams queryParams
+    userAgent = configApplicationName config <> " openapi3-code-generator/0.1.0.6 (https://github.com/Haskell-OpenAPI-Code-Generator/Haskell-OpenAPI-Client-Code-Generator)"
```

Comment:
We should not hardcode the version here, but instead have cabal provide it.

Reply:
You are aware that this is in the generated code, right (see `Embed.hs` on how we read the version from `package.yaml`)? I do not see why the version cannot be hard-coded in the generated code, as the code is only ever generated by one version of the generator. Could you elaborate on why this is a bad idea?